seed/utils/geocode.py

```diff
@@ -7,6 +7,7 @@
 
```

Comment:
Just noting that changes to this page are the relatively more impactful than other changes in this PR, and they deserve a closer review.